media-gfx/netgen/netgen-6.2.2302.ebuild

```diff
@@ -39,7 +40,7 @@ DEPEND="
 	jpeg? ( media-libs/libjpeg-turbo:0= )
 	logging? ( dev-libs/spdlog:= )
 	mpi? (
-		sci-libs/metis
+		<sci-libs/metis-5.2
```

Comment:
Is this still required with `-r2`?

Reply:
Just noticed the revbump after reviewing synced files. I will test if this is still necessary.